tests/base.conf.js

```diff
@@ -50,10 +51,17 @@ class BaseConfig {
         jasmine.getEnv().addReporter(utils.specReporter);
         jasmine.getEnv().addReporter(utils.reporter);
         browser.waitForAngularEnabled(false);
-
+        
+        // wait for startup to complete
         browser.driver.wait(startApi(), 135 * 1000, 'API took too long to start up');
-        browser.driver.sleep(10000); // wait for startup to complete
+        browser.driver.sleep(10000).then(() => {
+          if (pathToConfig){
```

Comment:
I don't think this needs to be in `then` - I think protractor will automatically pause execution until the sleep finishes.